resources/docs/cache-in-ci.md

```diff
@@ -12,7 +12,7 @@ return RectorConfig::configure()
         cacheClass: FileCacheStorage::class,
 
         // specify a path that works locally as well as on CI job runners
-        cacheDirectory: './var/cache/rector'
+        cacheDirectory: '/tmp/rector'
```

Comment:
using `/tmp/rector` as it works well for use.

phpstan by default uses `/tmp/phpstan`